### PR TITLE
Added Cudos token

### PR DESCRIPTION
**Pull request by diogofferreira**
CUDOS - The native token of the Cudos blockchain (https://www.cudos.org/)

### CODE REVIEW

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -1497,6 +1497,34 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/polkadot/images/dot.svg"
       },
       "coingecko_id": "polkadot"
+    },
+    {
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-1.assetlist.json:1501]: please remove listing from this file. It should only be in the frontier file for now.

Comment (diogofferreira) [reply osmosis-1/osmosis-1.assetlist.json]: @JeremyParish69 Done. For clarification purposes, how can we list the token on the main frontend?

Comment (JeremyParish69) [reply osmosis-1/osmosis-1.assetlist.json]: @diogofferreira To list on Osmosis Frontier, a PR is required to this frontend repo. E.g., https://github.com/osmosis-labs/osmosis-frontend/pull/723. To list on the Main Osmosis app, there should first be a Cudos pool with Osmosis bonding incentives, which must be approved by governance.

**Comment by IbrarMakaveli** [comment]
Add, thanks.
cf : https://github.com/osmosis-labs/assetlists/pull/237

**Pull request closed without merging**